model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -27,9 +27,15 @@ import kotlin.math.min
 import org.ossreviewtoolkit.model.CopyrightFinding
```

Comment:
Commit message: In the first sentence in the body something seems to be missing after "function", a "that"?